java/src/main/java/com/saucelabs/saucebindings/configs/SauceConfigsBrowser.java

```diff
@@ -0,0 +1,127 @@
+package com.saucelabs.saucebindings.configs;
+
+import com.saucelabs.saucebindings.PageLoadStrategy;
+import com.saucelabs.saucebindings.Prerun;
+import com.saucelabs.saucebindings.UnhandledPromptBehavior;
+import org.openqa.selenium.Proxy;
+
+import java.net.URL;
+import java.time.Duration;
+import java.util.Map;
+
+public class SauceConfigsBrowser<T extends SauceConfigsBrowser<T>> extends SauceConfigs<T> {
```

Comment:
What do you all think about calling this `BrowserConfiguration` and child classes `ChromeConfiguration`...?

Reply:
Yeah, I named them before I put them in a separate configs package, so I wanted them all grouped together...
Rethinking from big picture, this is probably how to break it up to best fit the capabilities spreadsheet:
Either Configs or Configurations is fine for me. It probably isn't the biggest issue either way since we theoretically don't want users to even know about the names of the classes. :)

Configs
----> VDCConfigs
--------> DesktopConfigs
------------> ChromeConfigs, EdgeConfigs, SafariConfigs, IEConfigs, FirefoxConfigs
--------> EmulatorConfigs, SimulatorConfigs
----> RDCConfigs
--------> AndroidConfigs, iOSConfigs